_includes/community/com-sql-query-run.md

```diff
@@ -0,0 +1,9 @@
+| Method | Description | Further information |
+|---|---|---|
+| CLI | Run SQL statements from the FeatureBase CLI system | [Enable SQL CLI](/docs/community/com-config/com-config-sql-cli-enable) |
+| FeatureBase API | FeatureBase provides an API which can be used to execute SQL statements | [Community API](/docs/community/com-api/com-api-home) |
+| Data import and ingest | Recreate table structure based on CSV, SQL or Kafka source files | [Import data](/docs/community/com-ingest/com-ingest-manage) |
+| HTTPS | Run SQL statements over HTTPS using the SQL CLI | [Enable SQL CLI](/docs/community/com-config/com-config-sql-cli-enable)
+| FBSQL CLI tool | SQL statements can be run via the FBSQL terminal tool | [FBSQL terminal tool](/docs/tools/fbsql/fbsql-home)|
```

Comment:
This is the same thing as the CLI above

Reply:
HTTPS is the same as the CLI tool -- I just put it on a separate line so it's scannable to find something specific.